seedvault/src/main/java/com/solanamobile/seedvault/PublicKeyResponse.java

```diff
@@ -72,7 +72,12 @@ public PublicKeyResponse(@Nullable byte[] publicKey,
     protected PublicKeyResponse(Parcel in) {
         mPublicKey = in.createByteArray();
         mPublicKeyEncoded = in.readString();
-        resolvedDerivationPath = in.readTypedObject(Uri.CREATOR);
```

Comment:
`readTypedObject` can return null, but `resolvedDerivationPath` is marked `@NonNull`